### ISSUE TITLE
Spotlight Indexer mds and Karabiner folder

### ISSUE THREAD

**Issue by kmplngj** [pqrs-org/Karabiner-Elements#2679]
Hello,

I just discovered that mds is constantly scanning the org.pqrs dir in /Library/Application Support/. Does anyone know why and how to fix this? This only happens with this folder.

Thanks
Jan

```08:56:33.611290  getattrlist                            /Library/Application Support/org.pqrs/tmp/user           0.000143   mds.46788
08:56:33.611325  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000034   mds.46788
08:56:33.611355  getattrlist                            /Library/Application Support/org.pqrs                    0.000029   mds.46788
08:56:33.611383  getattrlist                            /Library/Application Support                             0.000027   mds.46788
08:56:33.611409  getattrlist                            /Library                                                 0.000025   mds.46788
08:56:33.611618  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000023   mds.46788
08:56:33.611635  getattrlist                            /Library/Application Support/org.pqrs                    0.000015   mds.46788
08:56:33.611650  getattrlist                            /Library/Application Support                             0.000015   mds.46788
08:56:33.611665  getattrlist                            /Library                                                 0.000014   mds.46788
08:56:35.490180  read              F=5    B=0x43e                                                                2.071150   mds.4217
08:56:36.680098  getattrlist                            /Library/Application Support/org.pqrs/tmp/user           0.000082   mds.46788
08:56:36.680130  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000031   mds.46788
08:56:36.680159  getattrlist                            /Library/Application Support/org.pqrs                    0.000028   mds.46788
08:56:36.680181  getattrlist                            /Library/Application Support                             0.000021   mds.46788
08:56:36.680200  getattrlist                            /Library                                                 0.000018   mds.46788
08:56:36.680411  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000027   mds.46788
08:56:36.680432  getattrlist                            /Library/Application Support/org.pqrs                    0.000019   mds.46788
08:56:36.680447  getattrlist                            /Library/Application Support                             0.000015   mds.46788
08:56:36.680462  getattrlist                            /Library                                                 0.000014   mds.46788
08:56:38.501258  read              F=5    B=0x43e                                                                3.011042   mds.4217
08:56:38.537472    RdMeta[S]       D=0x00015e09  B=0x1000   /dev/disk3  /dev/disk3                               0.000777 W com.apple.WebKit.48722
08:56:39.327623  read              F=5    B=0x41c                                                                0.826335   mds.4217
08:56:39.328373  getattrlist                            /Library/Application Support/org.pqrs/tmp/user           0.000036   mds.46771
08:56:39.328386  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000012   mds.46771
08:56:39.328396  getattrlist                            /Library/Application Support/org.pqrs                    0.000010   mds.46771
08:56:39.328407  getattrlist                            /Library/Application Support                             0.000011   mds.46771
08:56:39.328416  getattrlist                            /Library                                                 0.000009   mds.46771
08:56:39.369785  read              F=5    B=0x549                                                                0.042148   mds.4217
08:56:39.435027  read              F=5    B=0x138                                                                0.065228   mds.4217
08:56:39.456785  read              F=5    B=0x4a2                                                                0.021737   mds.4217
08:56:40.633323  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000121   mds.46771
08:56:40.633347  getattrlist                            /Library/Application Support/org.pqrs                    0.000023   mds.46771
08:56:40.633374  getattrlist                            /Library/Application Support                             0.000027   mds.46771
08:56:40.633393  getattrlist                            /Library                                                 0.000018   mds.46771
08:56:41.559954  read              F=5    B=0x43e                                                                2.103149   mds.4217
08:56:42.462734  read              F=5    B=0x1e3                                                                0.902744   mds.4217
08:56:42.550414  read              F=5    B=0x13e                                                                0.087646   mds.4217
08:56:42.751611  getattrlist                            /Library/Application Support/org.pqrs/tmp/user           0.000106   mds.47144
08:56:42.751628  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000016   mds.47144
08:56:42.751648  getattrlist                            /Library/Application Support/org.pqrs                    0.000019   mds.47144
08:56:42.751666  getattrlist                            /Library/Application Support                             0.000017   mds.47144
08:56:42.751681  getattrlist                            /Library                                                 0.000015   mds.47144
08:56:42.751911  getattrlist                            /Library/Application Support/org.pqrs/tmp                0.000030   mds.47144
08:56:42.751961  getattrlist                            /Library/Application Support/org.pqrs                    0.000016   mds.47144
08:56:42.751974  getattrlist                            /Library/Application Support                             0.000013   mds.47144
08:56:42.751985  getattrlist                            /Library                                                 0.000010   mds.47144```

**Comment by tekezo** [comment]
Thank you for feedback!

I fixed this issue at v13.4.1.
Please confirm the latest beta version.

How to upgrade Karabiner-Elements to the latest beta version:
<https://karabiner-elements.pqrs.org/docs/manual/operation/check-for-updates/>

**Comment by kmplngj** [comment]
Thanks! Seems to be fixed.
